osbuild/pipeline.py

```diff
@@ -65,7 +64,7 @@ def run(self,
             libdir,
             var="/var/tmp"):
         with buildroot.BuildRoot(build_tree, runner, libdir, var=var) as build_root, \
-            tempfile.TemporaryDirectory(prefix="osbuild-sources-output-", dir=var) as sources_output:
+                tempfile.TemporaryDirectory(prefix="osbuild-sources-output-", dir=var) as sources_output:
```

Comment:
Was that intentional?

Reply:
No, somehow the linter - I'm still configuring. I'll fix it manually